src/commands/commerce/store/quickstart/setup.ts

```diff
@@ -290,6 +298,15 @@ export class StoreQuickstartSetup extends SfdxCommand {
             )} ServiceProviderType=${serviceProviderType}`,
             this.org.getUsername()
         );
+*/
+        const integrationName = `${serviceProviderType}'__'${developerName}`;
```

Comment:
why is integrationName being constructed here instead of retrieving the ID from the RegisteredExternalService which is the usual way to add integrations